enterprise/frontend/src/metabase-enterprise/api/upload-management.ts

```diff
@@ -0,0 +1,30 @@
+import { Api } from "metabase/api";
+import { provideTableListTags, invalidateTags, tag } from "metabase/api/tags";
+import type {
+  DeleteUploadTableRequest,
+  UploadManagementResponse,
+} from "metabase-types/api";
+
+export const uploadManagementApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listUploadTables: builder.query<UploadManagementResponse, void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/ee/upload-management/tables",
+      }),
+      providesTags: (tables = []) => provideTableListTags(tables),
+    }),
+    deleteUploadTable: builder.mutation<boolean, DeleteUploadTableRequest>({
+      query: ({ tableId, ...params }) => ({
+        method: "DELETE",
+        url: `/api/ee/upload-management/tables/${tableId}`,
+        params,
+      }),
+      invalidatesTags: (_, error) =>
+        invalidateTags(error, [tag("table"), tag("database")]),
```

Comment:
do we need to invalidate something field related here too?

Reply:
I think you might want to invalidate all the same things as `syncDatabaseSchema`?